contrib/init/sysvinit-redhat/docker

```diff
@@ -56,7 +56,7 @@ start() {
         prestart
         printf "Starting $prog:\t"
         echo "\n$(date)\n" >> $logfile
-        "$unshare" -m -- $exec daemon $other_args &>> $logfile &
+        "$unshare" -m -- $exec daemon $other_args & >> $logfile &
```

Comment:
`&>` is a bashism. Since the script uses `#!/bin/sh`, you can't rely on bash. However in redhat `/bin/sh` is bash, so you must have changed it somehow.

However the equivalent to `&>>` in `/bin/sh` (posix) would be `>> $logfile 2>&1`